Library Genesis.js

```diff
@@ -0,0 +1,245 @@
+{
+	"translatorID": "11614156-f421-4e89-8ce0-a5e69ce3ebed",
+	"label": "Library Genesis",
+	"creator": "Reverend Wicks Cherrycoke",
+	"target": "libgen.io|lib.gen.rus.ec",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsi",
+	"lastUpdated": "2016-04-22 22:11:28"
+}
+
+var MIME_TYPES = {
+  'pdf': 'application/pdf',
+  'epub': 'application/epub+zip',
+  'mobi': 'application/x-mobipocket-ebook',
+  'djvu': 'image/vnd.djvu'
+};
+var MD5_REGEX = /md5=([0-9a-fA-F]+)/;
+
+
+function detectWeb(doc, url) {
+  if (url.indexOf('book/index.php') != -1) {
+	return 'book';
+  }
+}
+
+function doWeb(doc, url) {
+  var md5Hash = MD5_REGEX.exec(url)[1];
+
+  // To save some work, we use the provided bibtex file to retrieve the
+  // metadata and use Zotero's built-in bibtex importer
+  var bibtexUrl = "http://libgen.io/book/bibtex.php?md5=" + md5Hash;
```

Comment:
use a relative url, i.e. `'/book/bibtex.php?md5=' + md5Hash;` Robust to https and uses the same page you're on.